lib/experimental-default-theme.json

```diff
@@ -1,227 +1,237 @@
+					"slug": "blush-bordeaux"
+				},
+				{
+					"name": "Luminous dusk",
+					"gradient": "linear-gradient(135deg,rgb(255,203,112) 0%,rgb(199,81,192) 50%,rgb(65,88,208) 100%)",
+					"slug": "luminous-dusk"
+				},
+				{
+					"name": "Pale ocean",
+					"gradient": "linear-gradient(135deg,rgb(255,245,203) 0%,rgb(182,227,212) 50%,rgb(51,167,181) 100%)",
+					"slug": "pale-ocean"
+				},
+				{
+					"name": "Electric grass",
+					"gradient": "linear-gradient(135deg,rgb(202,248,128) 0%,rgb(113,206,126) 100%)",
+					"slug": "electric-grass"
+				},
+				{
+					"name": "Midnight",
+					"gradient": "linear-gradient(135deg,rgb(2,3,129) 0%,rgb(40,116,252) 100%)",
+					"slug": "midnight"
+				}
+			],
+			"duotone": [
+				{
+					"name":  "Dark grayscale" ,
+					"colors": [ "#000000", "#7f7f7f" ],
+					"slug": "dark-grayscale"
+				},
+				{
+					"name":  "Grayscale" ,
+					"colors": [ "#000000", "#ffffff" ],
+					"slug": "grayscale"
+				},
+				{
+					"name":  "Purple and yellow" ,
+					"colors": [ "#8c00b7", "#fcff41" ],
+					"slug": "purple-yellow"
+				},
+				{
+					"name":  "Blue and red" ,
+					"colors": [ "#000097", "#ff4747" ],
+					"slug": "blue-red"
+				},
+				{
+					"name":  "Midnight" ,
+					"colors": [ "#000000", "#00a5ff" ],
+					"slug": "midnight"
+				},
+				{
+					"name":  "Magenta and yellow" ,
+					"colors": [ "#c7005a", "#fff278" ],
+					"slug": "magenta-yellow"
+				},
+				{
+					"name":  "Purple and green" ,
+					"colors": [ "#a60072", "#67ff66" ],
+					"slug": "purple-green"
+				},
+				{
+					"name":  "Blue and orange" ,
+					"colors": [ "#1900d8", "#ffa96b" ],
+					"slug": "blue-orange"
+				}
+			],
+			"custom": true,
+			"link": false,
+			"customGradient": true
+		},
+		"typography": {
+			"dropCap": true,
+			"customFontSize": true,
+			"customLineHeight": false,
+			"customFontStyle": true,
+			"customFontWeight": true,
+			"customTextTransforms": true,
+			"customTextDecorations": true,
+			"fontSizes": [
+				{
+					"name": "Small",
+					"slug": "small",
+					"size": "13px"
+				},
+				{
+					"name": "Normal",
+					"slug": "normal",
+					"size": "16px"
+				},
+				{
+					"name": "Medium",
+					"slug": "medium",
+					"size": "20px"
+				},
+				{
+					"name": "Large",
+					"slug": "large",
+					"size": "36px"
+				},
+				{
+					"name": "Huge",
+					"slug": "huge",
+					"size": "42px"
+				}
+			]
+		},
+		"spacing": {
+			"customPadding": false,
+			"units": [ "px", "em", "rem", "vh", "vw" ]
 		},
-		"core/button": {
-			"border": {
-				"customRadius": true
+		"border": {
+			"customColor": false,
+			"customRadius": false,
+			"customStyle": false,
+			"customWidth": false
+		},
+		"blocks": {
+			"core/button": {
+				"border": {
+					"customRadius": true
+				}
+			}
+		}
+	},
+	"styles": {
+		"elements": {
+			"link": {
+				"color": {
+					"text": "var(--wp--style--color--link, #00e)"
```

Comment:
Do we still need this `--wp--style--color--link` variable or is here for BC?

Reply:
This PR doesn't modify how user styles work. The link color in user-land still attaches the `--wp--style--color--link` to the wrapper, so we need to enqueue the rule `a { color: var(--wp--style--color--link, <theme-value>)}` somewhere for user changes to take effect. We previously did it in  `lib/global-styles.php` (see changes for that file in this PR).

I'm thinking we'd want to get rid of that variable by following what layout and duotone do. Thoughts?

A thought I have about this which makes me a bit uncomfortable is that this `link.color.text` is expected to be overridable by the theme's theme.json and user via global styles sidebar. This means that themes will have to set the text color of the link in this way: `var(--wp--style--color--link, <theme-color>)`, so the `--wp--style--color--link` is attached to the link color. If/when we change to follow layout/duotone footsteps this is no longer necessary.

Reply:
> I'm thinking we'd want to get rid of that variable by following what layout and duotone do. Thoughts?

Yes, this makes sense to me, we may want to keep it around for BC though, at least in the plugin somehow.